persistent-data-provider/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "commonjs",
+    "strict": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true
+  }
+}
```

Comment:
Do we wanna duplicate the tsconfigs from the main project?

Reply:
This `tsconfig` was autogenerated, so for now at least, I'd like to keep using this `tsconfig`. We can look into using the other `tsconfig` as a possible future modification.